src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Represents a storage class that contains methods to read from
+ * and write to the save file.
+ */
+public class Storage {
+    private String user_dir;
+    private DukeTaskList dukeTaskList;
+
+    /**
+     * Constructor of the Storage class.
+     *
+     * @param user_dir current directory where the user runs Duke from
+     * @param dukeTaskList a dukeTaskList object of the current run of Duke.
+     */
+    public Storage(String user_dir, DukeTaskList dukeTaskList) {
+        this.user_dir = user_dir;
+        this.dukeTaskList = dukeTaskList;
+    }
+
+    /**
+     * Method to read data from save file. If the save file and dir don't exist,
```

Comment:
Maybe you can write the first sentence of the header comment as 'Reads data from save file.' so that it more closely aligns with the coding standards for the module?

This can also be applied to some of the other methods in the code.